### PR TITLE
Fix mass delete for custom entities

### PR DESCRIPTION
**Pull request by fitn**

### CODE REVIEW

File: Entity/Repository/CustomEntityRepository.php [hunk 1]
```diff
@@ -13,4 +14,66 @@
  */
 class CustomEntityRepository extends ReferenceDataRepository
 {
+    /**
+     * Apply mass action parameters on query builder
```
Comment (rybus) [inline Entity/Repository/CustomEntityRepository.php]: Applies [...] on the

File: Entity/Repository/CustomEntityRepository.php [hunk 2]
```diff
@@ -13,4 +14,66 @@
...
+     *
+     * @param QueryBuilder $qb
+     * @param bool         $inset
+     * @param array        $values
+     */
+    public function applyMassActionParameters($qb, $inset, $values)
```
Comment (rybus) [inline Entity/Repository/CustomEntityRepository.php]: inSet?

Comment (fitn) [inline Entity/Repository/CustomEntityRepository.php]: It's done like that to be homogeneous with the PIM

Comment (rybus) [inline Entity/Repository/CustomEntityRepository.php]: ah ok :(

**Pull request merged**